nix/cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+function getSources() {
+    nix-store --query --binding src "${1}"
+}
+
+function getOutputs() {
+    nix-store --query --outputs "${1}"
+}
+
+function getDrvFiles() {
+    nix-store --query --deriver "${1}"
+}
+
+function getReferrers() {
+    nix-store --query --referrers "${1}"
+}
+
+toDelete=(
+  "prod-build-android"
+  "release-android"
+)
+
+for drvName in "${toDelete[@]}"; do
+    drvPaths=$(find /nix/store -type d -name "*${drvName}*")
+    echo "${drvPaths}" | while read path; do
+        echo " ? Checking:   ${path}"
+        drv=$(getDrvFiles "${path}")
+        # if drv is unknown-deriver then path is a source
+        if [[ "${drv}" == "unknown-deriver" ]]; then
+            drv=$(getReferrers "${path}")
+            src="${path}"
+        else
+            src=$(getSources "${drv}")
+        fi
+        echo " - Derivation: ${drv}"
+        echo " - Source:     ${src}"
+
+        echo "Deleting..."
+        echo nix-store --delete "${path}" "${drv}" "${src}"
```

Comment:
We could just build up a list of paths to delete and make a single call to `nix-store --delete` at the end.

Reply:
Yeah, I agree, I was mostly just trying to be explicit while developing.